_data/withshubh/withshubh.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Shubhendra Singh Chauhan
+institution: Jaypee University of Engineering and Technology 🚩 
+profile_pic: shubh.jpg
+quote: Thanks to those who believe in me. Advice for my juniors is that learn skills which interests them and enjoy their time while at university as its going to be the best 4 years of life that will never come back. 
```

Comment:
According to README.md, this quote can be no more than 100 characters. Your quote has zoomed past that limit at a wicked 88mph! 🚗🔥 If you could revise this to fit within the limit, that would be great!

From README.md:
> name: FULLNAME-OR-NICKNAME # No longer than 28 characters
> institution: INSTITUTION-NAME 🚩 # no longer than 58 characters
> profile_pic: mona.png # Name and extension of your profile picture(ex. mona.png) The picture must be squared and 544px on width and height.
> quote: YOUR-SENIOR-QUOTE # **no longer than 100 characters**, avoid using quotes(") to guarantee the format remains the same.
> github_user: YOUR-GITHUB-USERNAME